gaia-switch.js

```diff
@@ -44,11 +44,31 @@ module.exports = component.register('gaia-switch', {
     this.disabled = this.getAttribute('disabled');
     this.checked = this.getAttribute('checked');
 
+    // put non-critical operations in post-load time frame
+    // to save loadtime.
```

Comment:
According to research there's `50ms` post-load idle time when user see the web page is rendered. We could save overall loadtime by using that post-load idle time (user did not start interact with UI yet)  for non-critical operations.